content/announcements/news1.adoc

```diff
@@ -0,0 +1,6 @@
+---
+title: Test 1
+alerts: "alert"
+
+---
+Fedora" is not only a kind of hat, but also a large Open Source community with a unique role in the Open Source ecosystem. This page provides an overview of the Fedora Linux distribution, project, and community. hjgu ljgl kgto iu
```

Comment:
This file shouldn't be in this Pull Request yet. Could you please drop this one?